src/components/organisms/EmotionCard.tsx

```diff
@@ -0,0 +1,139 @@
+   * send the emotion to backend, and close the card.
+   * @param emotion Emotion that user select from the emotion card.
+   */
+  const handleEmotionSubmit = async (emotion) => {
+    await sendPayload(emotion);
+    await AsyncStorage.setItem(
+      'emotionDate',
+      JSON.stringify(moment().format('L'))
+    );
+    close();
+  };
+  const sendPayload = async (emotion) => {
+    const payload = {
+      emotionLevel: emotion
+    };
+    try {
+      const { data } = await client.post('/emotion-tracking', payload);
+      if (data) console.log('emotion sent: ' + emotion);
+    } catch (err) {
+      console.log('Unsuccessful emotion sent: ' + emotion);
+    }
+  };
+  /**
+   * This function get the greeting image according to the day of the week.
+   * @param date The day of the week
+   * @returns image path inside assets
+   */
+  const getImageSource = (date) => {
+    return getEmotionCardImage(date, language);
+  };
+  return (
+    <Modal isOpen={showEmotionCard} onClose={() => handleEmotionSubmit('NA')}>
+      <Modal.Content maxWidth="400px">
+        <Modal.CloseButton />
+        <Modal.Header style={styles.header}>
+          <Text fontSize={'lg'} fontWeight={'bold'}>
+            {t('emotionTrackingCard.' + message)}
+          </Text>
+        </Modal.Header>
+        <Modal.Body>
+          <View style={styles.pictureArea}>
+            <Image
+              width={'240 px'}
+              height={'320 px'}
+              source={getImageSource(message)}
+            />
+          </View>
+          <Text fontSize="lg" fontWeight="400">
+            {t('emotionTrackingCard.howAreYouFeelingToday')}
+          </Text>
+        </Modal.Body>
+        <Modal.Footer>
+          <View style={styles.emotionPicker}>
+            <TouchableOpacity
+              style={styles.emotionButton}
+              onPress={() => handleEmotionSubmit('HAPPY')}>
+              <Icon
+                as={MaterialCommunityIcons}
+                name="emoticon-happy-outline"
+                size="12"
+                color="black"
+              />
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.emotionButton}
+              onPress={() => handleEmotionSubmit('NEUTRAL')}>
+              <Icon
+                as={MaterialCommunityIcons}
+                name="emoticon-neutral-outline"
+                size="12"
+                color="black"
+              />
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.emotionButton}
+              onPress={() => handleEmotionSubmit('BAD')}>
+              <Icon
+                as={MaterialCommunityIcons}
+                name="emoticon-sad-outline"
+                size="12"
+                color="black"
+              />
+            </TouchableOpacity>
+          </View>
+        </Modal.Footer>
+      </Modal.Content>
+    </Modal>
+  );
+};
+
+export default EmotionCard;
+
+const styles = StyleSheet.create({
```

Comment:
You could write an in-line styling if you use native base.
For instance <View alignItems='center' ...>
(optional)